apps/ledger-live-desktop/src/renderer/components/TransactionConfirm/ConfirmAlert.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { useMemo } from "react";
+import { TFunction } from "i18next";
+import { Trans, withTranslation } from "react-i18next";
+import { Transaction } from "@ledgerhq/live-common/generated/types";
+import { DeviceTransactionField } from "@ledgerhq/live-common/transaction/index";
+import Alert from "~/renderer/components/Alert";
+
+type Props = {
+  t: TFunction;
+  transaction: Transaction;
+  typeTransaction: string | undefined;
+  fields: DeviceTransactionField[];
+};
+
+const ConfirmAlert = ({ t, transaction, typeTransaction, fields }: Props) => {
+  const key = ("mode" in transaction && transaction.mode) || "send";
```

Comment:
The ternary looks a bit easier to read
```suggestion
  const key = "mode" in transaction ? transaction.mode : "send";
```